luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/factories/AbridgedLogicalTableGroupFactory.java

```diff
@@ -0,0 +1,97 @@
+
+    /**
+     * Build a LogicalTable instance.
+     *
+     * @param name  the config dictionary name (normally the apiName)
+     * @param configTable  ObjectNode that points to the value of corresponding table entry in config file
+     * @param resourceFactories  the source for locating dependent objects
+     *
+     * @return  A newly constructed config instance for the name and config provided
+     */
+    @Override
+    public LogicalTableGroup build(String name, ObjectNode configTable, LuthierIndustrialPark resourceFactories) {
+        validateFields(name, configTable);
+
+        LogicalTableGroup tableDictionary = new LogicalTableGroup();
+        LogicalTableName tableName = LogicalTableName.forName(name);
+        MetricDictionary metricDictionary = resourceFactories.getMetricDictionary();
+        LinkedHashSet<Dimension> dimensions = new LinkedHashSet<>();
+        LinkedHashSet<PhysicalTable> physicalTables = new LinkedHashSet<>();
+        LinkedHashSet<ApiMetricName> apiMetricNames = new LinkedHashSet<>();
+        configTable.get("dimensions").forEach(
+                node -> dimensions.add(resourceFactories.getDimension(node.textValue()))
+        );
+        configTable.get("physicalTables").forEach(
+                node -> physicalTables.add(resourceFactories.getPhysicalTable(node.textValue()))
+        );
+        configTable.get("metrics").forEach(
+                node -> {
+                    /* go to the LIP to retrieve the metric */
+                    /* Must be done after having actual metric builders! */
+                    // apiMetricNames.add(new LuthierApiMetricName(node.textValue(), granularity));
+                    // metricDictionary.add();
+                }
+        );
+        TableGroup tableGroup = new TableGroup(
+                physicalTables,
+                apiMetricNames,
+                dimensions
+        );
+
+        configTable.get("granularities").forEach(
+                (JsonNode node) -> {
+                    ZonedTimeGrain granularity = DefaultTimeGrain.valueOf(
+                            node.textValue().toUpperCase(Locale.US)
+                    ).buildZonedTimeGrain(DateTimeZone.forID(configTable.get("dateTimeZone").textValue()));
+                    LogicalTable logicalTable = new LogicalTable(tableName, granularity, tableGroup, metricDictionary);
+                    tableDictionary.put(new TableIdentifier(logicalTable), logicalTable);
+                }
+        );
+
+        return tableDictionary;
+    }
+
+    /**
+     * makes sure the necessary fields exist in the json config.
+     *
+     * @param name  the config dictionary name (normally the apiName)
+     * @param configTable  ObjectNode that points to the value of corresponding table entry in config file
+     */
+    private void validateFields(String name, ObjectNode configTable) {
+        LuthierValidationUtils.validateField(configTable.get("granularities"), ENTITY_TYPE, name, "granularity");
```

Comment:
The last argument should be `"granularities"` not `"granularity"`.

Reply:
Yep.